ios-target-app/Onboarding/Views/SignUpViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SignUpViewController.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 6/09/22.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+
+    @IBOutlet weak var genderTextField: UITextField!
+    
+    @IBOutlet weak var ovalsGroup: UIImageView!
+    
+    let genders = ["Female", "Male", "Not defined"]
```

Comment:
are you planning to move the data out of the view controller?

Reply:
Oh not really, I'll make it private then.